streams/src/main/java/org/apache/kafka/streams/kstream/internals/suppress/SuppressedInternal.java

```diff
@@ -66,7 +79,7 @@ Duration timeToWaitForMoreEvents() {
     }
 
     boolean shouldSuppressTombstones() {
```

Comment:
Consider renaming to `safeToDropTombstones`:

```suggestion
     boolean safeToDropTombstones() {
```

Reply:
Ah, missed one. Thanks!